src/ReceivingList/ReceivingListFilter/ReceivingListFilter.test.js

```diff
@@ -77,13 +77,14 @@ describe('ReceivingListFilter', () => {
   it('should display receiving filters', async () => {
     renderReceivingListFilter();
 
-    expect(screen.getByText('ui-receiving.filter.orderStatus')).toBeDefined();
-    expect(screen.getByText('ui-receiving.filter.vendor')).toBeDefined();
-    expect(screen.getByText('ui-receiving.filter.orderType')).toBeDefined();
-    expect(screen.getByText('ui-receiving.filter.materialType')).toBeDefined();
-    expect(screen.getByText('ui-receiving.filter.orderFormat')).toBeDefined();
-    expect(screen.getByText('ui-receiving.filter.location')).toBeDefined();
-    expect(screen.getByText('ui-receiving.filter.rush')).toBeDefined();
+    expect(screen.getByText('ui-receiving.filter.orderStatus')).toBeInTheDocument();
```

Comment:
You don't even need `expect`; the `getBy()` methods throw if they don't find a match.